winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
+    var userForms = [ // TODO: state of all forms edited by user
```

Comment:
On the general idea, I would like to know out of forms [A,B,C,D,E], which forms have been edited by a user.

Maybe the user finished A and D and left C unfinished. Forms B and E are untouched.
So the list of ALL forms and the list of forms edited by a user is different.

In this case, `userForms` could be
```
[ { id: A, state: 'complete' }, { id: C, state: 'unfinished'}, { id: D, state: 'complete' } ]
```

`formTemplateService` gives us all forms that can be completed. But it doesn't give us which forms are completed, unfinished or untouched. The unfinished state is important because if I think it would be helpful to know if a form is partially completed, but was meant to be completed. From here, a user can discard it or edit it to complete the form. For now, since we only have one form, we can go ahead and just have two states: `completed` and `incomplete`.

But it seems to me `formTemplateService` is mainly for immutable values. We need a concept of mutable `state` values for each form.